src/doc/gui_manual/Operators/OperatorTypes/PICS/LimitCycle_operator.rst

```diff
@@ -0,0 +1,196 @@
+attributes common to all PICS operators, the Limit Cycle operator supports the
+following attributes.
+
+Integration Direction
+"""""""""""""""""""""
+
+Sets the integration direction through time. Options are:
+    
+Forward
+    Integrate forward in time.
+
+Backward
+    Integrate backward in time.
+
+Both
+    Integrate both forward and backward in time, producing two integral curves.
+
+Forward Directionless
+    Integrate forward in time assuming a directionless vector field.
+
+Backward Directionless
+    Integrate backward in time assuming a directionless vector field.
+
+Both Directionless
+    Integrate both forward and backward in time assuming a directionless vector
+    field, producing two integral curves.
+
+Eigen vectors are an example of a directionless vector field. To integrate
+using a directionless field requires that any orientation discontinuity be
+corrected prior to linear interpolation. That is, all vectors must be rotated 
+to match the orientation of the trajectory. The PICS code will do this
+processing for standard fields (e.g non-higher order elements).
+
+Integrator
+""""""""""
+
+See Integrator attributes that are common to all PICS operators.
+    
+Step Length
+"""""""""""
+
+See Step Length attributes that are common to all PICS operators.
+    
+Tolerances
+""""""""""
+
+See Tolerance attributes that are common to all PICS operators.
+
+Termination
+"""""""""""
+
+Integral curve termination can be controlled in several different ways. The
+termination is based on the most conservative criteria, so only one criteria
+must be met for termination. The options are:
+
+Maximum number of steps
+    The maximum number of integration steps that will be allowed.  
+
+
+Appearance
+^^^^^^^^^^
+
+The appearance tab specifies how the integral curve will be rendered. In addition
+to the Appearance attributes common to all PICS operators, the Integral Curve
+operator supports the following attributes.
+
+Cycle tolerance
+"""""""""""""""
+
+The smallest return distance for classifying an integral curve as a limit
+cycle.
+
+Maximum iterations
+""""""""""""""""""
+
+The maximum numbers of iterations when performing the bi-section method.
+
+Show partial results
+""""""""""""""""""""
+
+If the maximum number of bi-section iterations has been reached without finding
+a limit cycle show the integral curves still in the queue.
+
+Show the signed return distances for the first iteration
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Instead of plotting the limit cycles, plot the return distances along the
+Poincaré section.
+
+
+Data Value
+""""""""""
+
+Allows the user to set the data value associated with each data point on the
+integral curve. Options are:
+
+Solid
+    Each integral curve is given a single color.
+
+Seed ID
+    Each integral curve is given a different color.
+
+Speed
+    The magnitude of the vector field at each point.
+
+Vorticity Magnitude
+    The vorticity magnitude at each point.
```

Comment:
These and the other options (below) are worded slightly odd (as the one above that I suggested a revision of) so that they are not the clearest.